packages/react-server/src/ReactFlightOwnerStack.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
I think it's likely that `Fiber`/`Fizz`/`Flight` `OwnerStack.js` merge into a shared helper for extracting a presentable string.